katas/percy/multiples/mult.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```

Comment:
please refactor the code so it doesn't need to have a html file to be run, just nodejs